helios-client/src/main/java/com/spotify/helios/common/JobValidator.java

```diff
@@ -161,13 +161,10 @@ public JobValidator(final boolean shouldValidateJobHash,
         errors.add("Volume path is not absolute: " + path);
         continue;
       }
-      if (!isNullOrEmpty(source) && !source.startsWith("/")) {
-        errors.add("Volume source is not absolute: " + source);
-        continue;
-      }
```

Comment:
Should we still do some validation of `String source = entry.getValue()` ? For instance I'd expect that if it contained a `/`, it would need to be an absolute path. I'd imagine that for the other types of volumes a `/` would never be allowed (although trying to reimplement docker's validation logic would be overkill)